contrib/chart/backstage/templates/ingress.yaml

```diff
@@ -1,7 +1,13 @@
 {{- $frontendUrl := urlParse .Values.appConfig.app.baseUrl}}
 {{- $backendUrl := urlParse .Values.appConfig.backend.baseUrl}}
 {{- $lighthouseUrl := urlParse .Values.appConfig.lighthouse.baseUrl}}
+
+{{/* Determine the api type for the ingress */}}
+{{- if lt .Capabilities.KubeVersion.Minor "19" }}
 apiVersion: networking.k8s.io/v1beta1
+{{- else if gt .Capabilities.KubeVersion.Minor "19" }}
```

Comment:
wait, does this really work when it's _exactly_ 19? :) isn't there a plain `else` to use?